x-pack/plugins/cases/server/routes/api/internal/find_assignees.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UserProfileService } from '../../../services';
+import { INTERNAL_FIND_ASSIGNEES_URL } from '../../../../common/constants';
+import { createCaseError } from '../../../common/error';
+import { createCasesRoute } from '../create_cases_route';
+
+export const findAssigneesRoute = (userProfileService: UserProfileService) =>
+  createCasesRoute({
+    method: 'get',
+    path: INTERNAL_FIND_ASSIGNEES_URL,
+    routerOptions: {
+      tags: ['access:casesFindAssignedUsers'],
```

Comment:
I added a new tag since the `_find_assignees` route is different from the `_suggest_user_profiles`. Each time we add a new one we have to add it in all the various plugins though.